doc/Possible-key-values.md

```diff
@@ -48,7 +37,18 @@ A literal tab character, which is different from `tab` in certain apps. | `\t`
 
 The characters `?`, `#`, and `@` do not need to be escaped when writing custom layouts. Internally, they can be escaped by prepending backslash (by typing `\?`, `\#`, and `\@`).
 
-The characters `,` and `:` can be escaped in a key value, using single quotes. For example, this macro defines a key with legend `http` that sends a string containing `:`: `<key c="http:home,'https://'" />` For simplicity, `,` and `:` cannot be escaped in the key legend.
+The characters `,` and `:` must be escaped in a key value, using single quotes.
+
+**When defining a legend**, there are no escape codes.
+
+### Examples
```

Comment:
I don't think adding more examples is useful. At least a few selected examples should be closer to the definition of what is a key, as it was before.

I wouldn't be against a large "examples" section, that could give customization ideas. But not part of what defines a key.